_typos.toml

```diff
@@ -0,0 +1,3 @@
+[default.extend-words]
+thur = "thur"
```

Comment:
where is the "thur" used in this PR?

Reply:
It's the whitelist to inform `typos` that `thur` is a valid spell, for `Thur.`

Reply:
thx for the explanation, I didn't see the word "thur" in the diff view in this PR, so this word was appended to whitelist because it is used in `neuron-ui`, right?

Reply:
Ok, I've found it in i18n file